### PR TITLE
GLT-2182: report HiSeq and NovaSeq run positions

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/DefaultIllumina.java [hunk 1]
```diff
@@ -89,8 +91,12 @@
       XPathExpression miSeqPartNumber = xpath.compile("//FlowcellRFIDTag/PartNumber/text()");
       XPathExpression nextSeqPartNumber = xpath.compile("//FlowCellRfidTag/PartNumber/text()");
       XPathExpression novaSeqPartNum = xpath.compile("//RfidsInfo/FlowCellMode/text()");
-
       CONTAINER_PARTNUMBER_XPATHS = Collections.unmodifiableSet(Sets.newHashSet(miSeqPartNumber, nextSeqPartNumber, novaSeqPartNum));
+      
+      XPathExpression hiSeqPosition = xpath.compile("//Setup/FCPosition/text()");
```
Comment (apmasell) [inline scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/DefaultIllumina.java]: I don't know if this is more or less awful, but all of these XPath expression could be joined:
```
XPathExpression novaSeqPosition = xpath.compile("//Setup/FCPosition/text()|//Side/text()");
```

and then it doesn't have to be a collection.

Comment (callunity) [reply scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/DefaultIllumina.java]: I like keeping things separate so we can see which instrument models have which changes

**Comment by callunity** [comment]
Oops sorry for not reviewing this before the release. I spoke with @apmasell and he said he'd release this next Tuesday when he's doing the MISO release, so that things can be properly tested on staging.

File: scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/DefaultIllumina.java (hunk 1 above)
Comment (djcooke) [reply scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/DefaultIllumina.java]: Yeah, I think it's more maintainable with them kept separate

**Pull request merged**